3Dmol/glviewer.js

```diff
@@ -4548,6 +4548,48 @@ $3Dmol.GLViewer = (function() {
             config.cartoonQuality = val;
         };
 
+        // State Management function 
+        /**
+         * Calls StateManager to add selection and style on the ui
+         * @function $3Dmol.GLViewer#loadSelectionStyle
+         * @param {AtomSelectionSpec} sel Atom Selection spec
+         * @param {AtomStyleSpec} style Atom Style spec
+         */
+        this.loadSelectionStyle = function(sel, style){
```

Comment:
Please make these more clearly related to the UI, otherwise the names are confusing.   I would create a `ui` object within the viewer so the call would be `glviewer.ui.loadSelectionStyle`

Reply:
Added `.ui` in the glviewer